src/stash.rs

```diff
@@ -258,4 +321,40 @@ mod tests {
 
         assert!(stash_name.starts_with("WIP on main:"));
     }
+
+    fn create_file(r: &Repository, name: &str, data: &str) -> PathBuf {
+        let p = Path::new(r.workdir().unwrap()).join(name);
+        fs::File::create(&p)
+            .unwrap()
+            .write(data.as_bytes())
+            .unwrap();
```

Comment:
This can be written a little more briefly as:

```suggestion
        fs::write(&p, data).unwrap();
```

I realize you're probably just copying what is done above. Feel free to update the tests above to use this function, too.